event-dispatcher/src/main/java/io/sk8s/event/dispatcher/FunctionDeployer.java

```diff
@@ -144,22 +158,20 @@ private VolumeMount buildNamedPipesMount() {
 	}
 
 	private EnvVar[] buildSidecarEnvVars(XFunction function) {
-		Map<String, String> springApplicationConfig = new HashMap<>();
-
-		springApplicationConfig.put("spring.cloud.stream.bindings.input.destination", function.getSpec().getInput());
-		springApplicationConfig.put("spring.cloud.stream.bindings.input.group", function.getMetadata().getName());
-		springApplicationConfig.put("spring.cloud.stream.bindings.output.destination", function.getSpec().getOutput());
-		springApplicationConfig.put("spring.cloud.stream.kafka.binder.brokers",
-				System.getenv("SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS"));
-		springApplicationConfig.put("spring.cloud.stream.kafka.binder.zkNodes",
-				System.getenv("SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES"));
-		springApplicationConfig.put("spring.application.name","sidecar-" + function.getMetadata().getName());
-		springApplicationConfig.put("server.port", "7433");
-		springApplicationConfig.put("spring.profiles.active",  function.getSpec().getProtocol());
+		Map<String, Object> config = new LinkedHashMap<>();
+		config.put("spring.cloud.stream.bindings.input.destination", function.getSpec().getInput());
+		config.put("spring.cloud.stream.bindings.output.destination", function.getSpec().getOutput());
+		config.put("spring.cloud.stream.bindings.input.group", function.getMetadata().getName());
+		config.put("spring.profiles.active", function.getSpec().getProtocol());
+		config.put("spring.application.name", "sidecar-" + function.getSpec().getInput());
+
+		config.put("spring.cloud.stream.kafka.binder.brokers", System.getenv("SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS"));
+		config.put("spring.cloud.stream.kafka.binder.zkNodes", System.getenv("SPRING_CLOUD_STREAM_KAFKA_BINDER_ZK_NODES"));
+		config.put("server.port", "-1");
```

Comment:
we might as well use a specific port for the sidecar, as long as it doesn't conflict with the function container port (8080)... 7433 mapped to "side" on a telephone, but we could use 8081 or whatever

Reply:
I used -1 to disable entirely (should that be 0?).

Reply:
I see. You're right: -1 disables (0 would be random). I think we would use `/health` for a readiness probe though. Is there a reason you included this in this PR? If not, let's treat it as a separate concern. If we truly want to disable, then we should avoid using a `WebApplicationContext` in the first place (and find an alternative for a readiness probe).

Reply:
Note that the go sidecar doesn't (yet) create an http server, and we don't set up probes. So wouldn't it be better to disable for now, and re-enable when everything works together?

Reply:
And to answer your question, this was a first attempt at speeding up the java sidecar

Reply:
ok, then - fine to disable for now. Let's capture an issue to either switch to the Go Sidecar or optimize the Java Sidecar. We'd want an issue for adding zipkin tracing to the Go Sidecar as well.

Reply:
Created https://github.com/ericbottard/function-sidecar/issues/2 and https://github.com/ericbottard/function-sidecar/issues/3